hub/api/dataset.py

```diff
@@ -262,11 +267,14 @@ def pytorch(self, transform: Optional[Callable] = None, workers: int = 1):
         Args:
             transform (Callable, optional) : Transformation function to be applied to each sample
             workers (int): The number of workers to use for fetching data in parallel.
+            tuple_fields (List, optional): If given, each sample will be a tuple in the specified order.
```

Comment:
Can we reword this to make this more clear? Someone without context might find it confusing.

Reply:
Any suggestions?

Reply:
```suggestion
            tuple_fields (List, optional): Optionally provide a list of tensor names in the ordering that your training script expects. For example, if you have a dataset that has "image" and "label" tensors, if `tuple_fields=["image", "label"]`, your training script should expect each batch will be provided as a tuple of (image, label).
```

Reply:
can you rename this variable to `tensors`?